other/bootstrap_daemon/tox_bootstrap.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Tox DHT Bootstrap Daemon
+After=network.target
+
+[Service]
+Type=forking
+PermissionsStartOnly=true
+ExecStartPre=/usr/bin/touch /var/run/tox-bootstrapd.pid
+ExecStartPre=/usr/bin/chown tox-bootstrapd:tox-bootstrapd -R /var/run/tox-bootstrapd.pid
```

Comment:
Debian has `chown` in `/bin`, not in `/usr/bin/`, so it fails for me. Also, the config file tells daemon to create PID file at `/var/run/tox-bootstrapd/tox-bootstrapd.pid`, which also fails since you use different path here. I think that using `/var/run/tox-bootstrapd.pid` is better, so the config and .sh files should be fixed to use that.